js/ui/resizable.js

```diff
@@ -53,9 +53,8 @@ var Resizable = DOMComponent.inherit({
 
             /**
             * @name dxResizableOptions.handles
-            * @type string
+            * @type Enums.ResizeHandle
```

Comment:
Ok, I think, we can add string here to avoid BC in TS
`* @type Enums.ResizeHandle | string`

Reply:
@lukyanovas what kind of BC do you expect on TS side?
https://github.com/DevExpress/DevExtreme/blob/18_2/ts/dx.all.d.ts#L3493

Reply:
Hm, I think this is a bug.

Reply:
It would be great to add string type right now to fix all problems with this case for a long time (perhaps, until we introduce "combination" type.)

Reply:
Ok, we will check `@type Enums.ResizeHandle | string` on SMDG side...